src/bitboxbase/bitboxbase_background.c

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Shift Cryptosecurity AG
```

Comment:
move into bitboxbase/components?

Reply:
I'll do that if/when we have more components. Sounds a bit overkill for 2-3 files.